src/game/shell.c

```diff
@@ -182,6 +187,78 @@ void Shell_Wait(int nticks)
     }
 }
 
+void Shell_GetScreenshotName(char *str)
+{
+    // Get level title of unknown length
+    char level_title[100];
+    sprintf(level_title, "%s", g_GameFlow.levels[g_CurrentLevel].level_title);
```

Comment:
Nitpick: this doesn't check for `LEVEL_TITLE_SIZE`

Reply:
The game crashes on screenshot if the user provides a title longer than `char level_title[100]` size in the gameflow title because I removed the gameflow validate / sanitize function. So I made this a longer buffer just in case and then trim it to `LEVEL_TITLE_SIZE` for the screenshot. That's why I put in the comment that the level title is unknown length.

Reply:
I meant using `snprintf` or similar. `sprintf` will copy the entire thing and may go over 100 characters.